GOVERNANCE.md

```diff
@@ -1,132 +1,175 @@
+- Esther Jackson [@ej2432](https://github.com/ej2432) and on the [forum](https://forum.openrefine.org/u/ej2432/summary)
 
 ### Project Manager
-OpenRefine's Project Manager (paid position) works closely with OpenRefine’s advisory and steering committees and with its communities of users and contributors.
+OpenRefine's Project Manager (paid position) works closely with OpenRefine's advisory committee, Core Dev Groups and OpenRefine users and contributors.
 
 The Project Manager:
-* Helps to improve OpenRefine's governance and community diversity
-* Helps build community-driven structures to formalize OpenRefine’s roadmap and keep it up to date
-* Assists in finding and secure new sources of funding
-* Supports the Advisory Committee in OpenRefine’s day-to-day operations.
+- Helps to improve OpenRefine's governance and community diversity
+- Helps build community-driven structures to formalize OpenRefine's roadmap and keep it up-to-date
+- Assists in finding and securing new sources of funding
+- Supports the Advisory Committee in OpenRefine's day-to-day operations; this may include supervising paid project staff.
 
-The current Project Manager is [Martin Magdinier](https://forum.openrefine.org/u/Martin/summary)
-## Fiscal sponsorship: Code For Science and Society
+The current Project Manager is Martin Magdinier [@magdmartin](https://github.com/magdmartin)and on the [forum](https://forum.openrefine.org/u/Martin/summary).
 
-Since January 2020, OpenRefine is a member project of [Code For Science and Society](https://codeforscience.org/) (CS&S).
+## Member Election
+When voting for new Committers, Core Developer Group or Code of Conduct Committee members, start a new thread with a subject line starting with `[VOTE]` to indicate that a formal vote has been requested on the [developer discussion list](https://forum.openrefine.org/c/dev/8) and use the `election` tag. In your message, indicate
+* The name of the individual nominated with its GitHub or forum username
+* For which group you are nominating this person (Committers, Core Developer or Code of Conduct Committee)
+* Why are you nominating this person? You can list significant contributions and adherence to OpenRefine governance.
 
-CS&S provides administrative and fiscal infrastructure to receive and manage funds under the CS&S Fiscal Sponsorship Agreement. CS&S is a USA-based organization qualified as exempt from federal income tax under Section 501(c)(3) of the Internal Revenue Code (IRC) and classified as a public charity under IRC Sections 509(a)(1) and 170(b)(1)(A)(vi).
+See, for example [this message](https://forum.openrefine.org/t/nominating-sunilnatraj-to-be-part-of-the-openrefine-github-organization/1914)
 
-### Manage funds
+Voting is open for 72 hours to provide an opportunity for all concerned persons to participate, regardless of their geographic location.
```

Comment:
```suggestion
Voting is open for seven days to provide an opportunity for all concerned persons to participate, regardless of their geographic location.
```

Increasing to seven days as per @thadguidry  request. Based on how frequently a number of committers and core contributors interact, I think it makes sense to increase it so we do not create artificial pressure to vote.